apps/src/templates/instructions/RubricField.jsx

```diff
@@ -55,27 +62,47 @@ class RubricField extends Component {
     currentlyChecked: PropTypes.bool
   };
 
+  constructor(props) {
+    super(props);
+    this.tooltipId = _.uniqueId();
```

Comment:
you can move this line into `render()` and remove the constructor. that way, tooltipId also won't need to be set at the class/component level (i.e., you'll set `const tooltipId = ...` instead of `this.tooltipId = ...`)